### PR TITLE
Gh actions edge

### PR DESCRIPTION
**Pull request by martialonline**
### Proposed changes
Add edge pipeline workflow, use latest Kubernetes (1.19.0), update Helm template RBAC endpoint to v1.

### CODE REVIEW

**Review by tellet — APPROVED** [review]
Lgtm

File: deployments/helm-chart/templates/rbac.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {{- if .Values.rbac.create }}
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (Dean-Coakley) [inline deployments/helm-chart/templates/rbac.yaml:3]: Can you make the same change?:
https://github.com/nginxinc/kubernetes-ingress/blob/gh-actions-edge/deployments/rbac/rbac.yaml#L2

**Review by Dean-Coakley — APPROVED** [review]
LGTM. Pending apiVersion upgrade for rbac manifests.

File: deployments/helm-chart/templates/rbac.yaml (hunk 1 above)
Comment (martialonline) [reply deployments/helm-chart/templates/rbac.yaml:3]: Done, thanks!

**Pull request merged**